### PR TITLE
firefox bug + feedback/apply section

### LINKED ISSUE #156: Firefox bug + Feedback + join core section desktop + mobile
- [ ] Fix the below issue with firefox on website and live site (at certain breakpoints the entire background is white) 
Screenshot 2023-02-03 at 4 51 31 PM

- [ ] Implement this section for desktop + mobile on the live site 
Screenshot 2023-02-03 at 4 52 07 PM
Screenshot 2023-02-03 at 4 52 36 PM

- [ ] For the feedback link, create a form like this https://forms.gle/2c8ZdPHBMAbC3UuA7 (last years) and put it in the day-of folder of Event Logistics
- [ ] For the apply link, use this link https://docs.google.com/forms/d/e/1FAIpQLScyEQzS4RBkkuPSC4cRlVFUYqnu0mkAcC4ypA21aR4aC2C6LQ/viewform

### PR DESCRIPTION
**Pull request by deanframe**
<Summary line - One sentence describing your intended set of changes>
Fixes firefox background bug, and adds feedback/apply section for desktop and mobile

Fixes #156 .

Changelist:

- fixes firefox background bug
- adds feedback / apply section with updated form
- makes welcome section text white when in desktop resolution and in night mode, since brown text against a navy blue background is hard to read
- fixes console errors
- code formatting

Tested:

- locally chrome + firefox on windows 10

### CODE REVIEW

File: live-site/components/feedback-join-core-section/FeedbackJoinCoreSection.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  gap: 2em;
+  flex: 100%;
+
+  @media ${min.tablet} {
+    flex: 50%;
+  }
+`;
+
+const StyledColumnTitle = styled.div`
+  font-size: 36px;
```
Comment (jzhzhang66) [inline live-site/components/feedback-join-core-section/FeedbackJoinCoreSection.styles.tsx]: em

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
LOOKIN GOOD DEAN!!! 

Just a few small things:

Could you please make both forms public? (or ask sreya or whoever is in charge of them to) 
Screenshot 2023-02-07 at 2 18 48 PM

I still see the firefox bug at some smaller screensizes but if its too hard to fix don't worry about it since most people don't use firefox on their phonees anyways haha
Screenshot 2023-02-07 at 2 20 00 PM

**Comment by deanframe** [comment]
@jzhzhang66 should be good now! other than the apply form which sreya will update

**Review by jzhzhang66 — APPROVED** [review]
LGTM just fix the white text thing!!! TY

**Pull request merged**